charts/fluentd/templates/daemonset.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         k8s-app: {{ .Values.k8sApp }}
         version: v1
     spec:
+      priorityClassName: {{ .Values.daemonset.priorityClassName }}
```

Comment:
Please add a condition that adds this field if the value is set.

Reply:
Fixed